src/core/connection.c

```diff
@@ -793,7 +793,7 @@ QuicConnUpdateRtt(
     if (RttUpdated) {
         CXPLAT_DBG_ASSERT(Path->SmoothedRtt != 0);
         QuicTraceLogConnVerbose(
-            RttUpdated,
+            RttUpdatedMsg,
```

Comment:
Instead of `Msg` what about `Evt`?

Reply:
feel free to send a commit;  I like Msg better :)